tasks/metrics/ci.rake

```diff
@@ -3,6 +3,7 @@ task ci: %w[ci:metrics metrics:mutant]
 
 namespace :ci do
   tasks = %w[
+    spec:integration
```

Comment:
I hate having to thing sequentially. I want all this to happen in parallel. But yeah I think you are right integration tests should come after the fast feedback tasks.